modules/connectivity-map-output-formats-and-visualizations.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * operating/build-time-network-policy-tools.adoc
+
+:_content-type: CONCEPT
+[id="connectivity-map-output-formats-and-visualizations_{context}"]
+= Connectivity map output formats and visualizations
+
+You can use various output formats, including `txt`, `md`, `csv`, `json`, and `dot`. The `dot` format, which is ideal for visualizing the output as a connectivity graph, can be viewed using the link:https://graphviz.org/[Graphviz tool]. You can convert the `dot` output to formats such as `svg`, `jpeg` or `png` using Graphviz, whether it is installed locally or through an online viewer.
```

Comment:
^I'm not sure what specifically Red Hat's policy is, but we probably want to word it this way so that it doesn't imply that you have to use this specific tool. (I'm assuming any graph visualization tool is OK.)